src/vs/platform/issue/electron-main/issueMainService.ts

```diff
@@ -442,6 +442,26 @@ export class IssueMainService implements IIssueMainService {
 		});
 	}
 
+	async $sendReporterStatus(extensionId: string, extensionName: string): Promise<boolean[]> {
+		const window = this.issueReporterWindowCheck();
+		const replyChannel = `vscode:sendReporterStatus`;
+		return Promises.withAsyncBody<boolean[]>(async (resolve) => {
+			const cts = new CancellationTokenSource();
+			const result = [false, false];
+			window.sendWhenReady(replyChannel, cts.token, { replyChannel, extensionId, extensionName });
+			validatedIpcMain.on('vscode:receiveReporterStatus', (_: unknown, data: boolean[]) => {
```

Comment:
Don't you wanna do `once` here?

https://github.com/microsoft/vscode/blob/f49bc367f1efe0e1dde9d85b2fdea5e65c683b5d/src/vs/base/parts/ipc/electron-main/ipcMain.ts#L45